src/main/java/Main.java

```diff
@@ -0,0 +1,47 @@
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        List<Car> carList = inputCar(scanner);
+        int time = inputTime(scanner);
+
+        RacingCars game = new RacingCars(carList);
+        game.playRound(time)
+                .printWinner();
+    }
+
+    public static List<Car> inputCar(Scanner scanner) {
+        System.out.println("경주할 자동차 이름을 입력하세요.");
+        String input = scanner.nextLine();
+        if (input.isEmpty()) {
+            throw new IllegalArgumentException();
+        }
+        return GenerateCar(input);
+    }
+
+    private static List<Car> GenerateCar(String input) {
+        String[] names = input.split(",");
+        List<Car> cars = new ArrayList<>();
+        for (String name : names) {
+            cars.add(new Car(name, new RandInt()));
+        }
+        return cars;
+    }
+
+    public static int inputTime(Scanner scanner) {
+        System.out.println("경기횟수를 입력하세요");
+        int time = 0;
+        try {
+            time = scanner.nextInt();
+        } catch (InputMismatchException e) {
+            System.out.println("숫자가 아닙니다");
+            throw e;
+        }
+        return time;
+    }
```

Comment:
`time`이라는 변수를 만들고, 재할당하고 있네요!
그런데 굳이 변수를 만들고, 재할당 할 필요가 있을까요?

```suggestion
    public static int inputTime(Scanner scanner) {
        System.out.println("경기횟수를 입력하세요");
        try {
            return scanner.nextInt();
        } catch (InputMismatchException e) {
            System.out.println("숫자가 아닙니다");
            throw e;
        }
    }
```

Reply:
변수를 만들고 할당을 하여 return을 하는거에 익숙해져있었는데 이렇게 코드를 더 간결하게 만들 수 있겠네요 최대한 변수 생성은 자제하는 쪽으로 코드를 작성해보겠습니다! 감사합니다